### PR TITLE
feat(directive): Added component creation event

### PR DESCRIPTION
**Pull request by chriswhite199**
To Address #135

### CODE REVIEW

File: src/dynamic/dynamic.component.spec.ts [hunk 1]
```diff
@@ -60,6 +62,13 @@ describe('DynamicComponent', () => {
         expect(injectedElem.componentInstance).toEqual(jasmine.any(InjectedComponent));
     });
 
+    it('should emit event when component created', () => {
+        fixture.componentInstance.component = InjectedComponent;
+        fixture.detectChanges();
+
+        expect(fixture.componentInstance.comp.instance).toBeInstanceOf(InjectedComponent);
```
[the remark this answers is not in the document]
Comment (chriswhite199) [reply src/dynamic/dynamic.component.spec.ts]: Would adding the following be more a exhaustive test / expectation:

    expect(fixture.componentInstance.comp.instance).toBeTruthy();

[the remark this answers is not in the document]
Comment (gund) [reply src/dynamic/dynamic.component.spec.ts]: I'm sorry, you are right. I just was expecting some `jest.fn()` mocks to be in the test suits =)

**Comment by gund** [comment]
LGTM

Thanks for making a PR =)

**Pull request merged**